src/pet.js

```diff
@@ -0,0 +1,75 @@
+const initialAge = 0;
+const deathAge = 30;
+
+const initialHunger = 0;
+const deathHunger = 10;
+
+const initialFitness = 10;
+const deathFitness = 0;
+
+const getFitBy = 4;
+const loseFitnessBy = 3;
+const getHungryBy = 5;
+const decreaseHungerBy = 3;
+const getOlderBy = 1;
+
+class Pet {
+  constructor(petName) {
+    this.name = petName;
+    this.age = initialAge;
+    this.hunger = initialHunger;
+    this.fitness = initialFitness;
+  }
+  get isAlive() {
+    return (
+      this.fitness > deathFitness &&
+      this.hunger < deathHunger &&
+      this.age < deathAge
+    );
+  }
+
+  growUp() {
+    if (!this.isAlive) {
+      throw new Error("Your pet is no longer alive :(");
```

Comment:
This message is referenced several times in the file, could it be put into a constant along with the values?